src/inspector_agent.cc

```diff
@@ -380,6 +388,53 @@ void AgentImpl::WaitForDisconnect() {
   inspector_->runMessageLoopOnPause(0);
 }
 
+String16 toProtocolString(v8::Local<v8::Value> value) {
+  if (value.IsEmpty() || value->IsNull() || value->IsUndefined() ||
+      value->IsString()) {
+    return String16();
+  }
+  v8::Local<v8::String> string_value = v8::Local<v8::String>::Cast(value);
+  std::unique_ptr<UChar[]> buffer(new UChar[string_value->Length()]);
```

Comment:
You probably cannot use std::unique_ptr as long as we still support OS X 10.8 and 10.9.

(I'm not 100% sure about 10.9 but 10.8 certainly doesn't have it, not even in tr1.)